integration-tests/src/main/java/io/quarkus/it/amazon/dynamodb/enhanced/DynamoDbEnhancedResource.java

```diff
@@ -88,4 +91,40 @@ public String testBlockingDynamo() {
             return "ERROR";
         }
     }
+
+    @GET
+    @Path("blocking/extensions")
```

Comment:
What is the difference between paths blocking/extensions and blocking? You mentioned that VersionRecordExtension is enabled by default so existingTableEntry.getCreatedAtTimestamp() should also work in testBlockingDynamo() shouldn't it? And where is blocking/extensions called in the tests?

Reply:
> What is the difference between paths `blocking/extensions` and `blocking`? You mentioned that `VersionRecordExtension` is enabled by default so `existingTableEntry.getCreatedAtTimestamp()` should also work in `testBlockingDynamo()` shouldn't it? And where is blocking/extensions called in the tests?

I don't know dynamodb that much but I guess `VersionRecordExtension` increment a counter property annotated with a associated annotation while `AutoGeneratedTimestampRecordExtension` is the one that actually set DateTime properties.

Reply:
Oh, didn't see that `AutoGeneratedTimestampRecordExtension` is set in application.properties. Okay, got it thanks. Nevertheless in [AmazonDynamoDBEnhancedTest.java](https://github.com/quarkiverse/quarkus-amazon-services/pull/488/files#diff-7f09dd446cd33221d3ee77864353414406dfff169043e9c2e66a1b6282415b78) I only see calls to /async and /blocking but no /blocking/extensions. Maybe I am missing something, it's late here.